utils/crash-error-report.js

```diff
@@ -0,0 +1,9 @@
+const eventtracking = require('./eventtracking');
+
+const mixPanelProperties = {
+    near_shell_command: process.env.NEAR_CLI_ERROR_LAST_COMMAND,
+    error_message: process.env.NEAR_CLI_LAST_ERROR,
+    network_id: process.env.NEAR_CLI_NETWORK_ID
+};
+Object.assign(mixPanelProperties, {});
```

Comment:
I don't understand this line, isn't this adding nothing to the `mixPanelProperties`?
Looking at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Reply:
tested this by removing this line and it works, I think we can get rid of it

Reply:
This was a copy paste artifact, thanks for catching!